### PR TITLE
Add KNOWNBUG for multidimentional char array[TG-8466]

### PR DESCRIPTION
**Pull request by yumibagge**
- [x] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- n/a Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: jbmc/regression/jbmc/multidimensional-array-char/test.desc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+My.class
+--function My.check2DCharArray --unwind 5 --max-nondet-array-length 10
+line 8 assertion at file My.java.*: FAILURE
+line 11 assertion at file My.java.*: FAILURE
+^VERIFICATION FAILED$
+^EXIT=10$
+^SIGNAL=0$
+--
+--
+Multidimentional char array used to cause a performance issue (test-gen 0.8.1).
```
Comment (romainbrenguier) [inline jbmc/regression/jbmc/multidimensional-array-char/test.desc]: I don't think it make sens to mention test-gen version here

File: jbmc/regression/jbmc/multidimensional-array-char/My.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        if (board[row][col] == 'O') {
+          diff++;
+          assert !assertionCheck;
+        } else if (board[row][col] == 'X') {
+          diff--;
+          assert !assertionCheck;
+        }
+      }
+    }
+    return 0;
```
Comment (romainbrenguier) [inline jbmc/regression/jbmc/multidimensional-array-char/My.java]: :pick: doesn't really matter, but I would prefer if the function was returning `diff`

File: jbmc/regression/jbmc/multidimensional-array-char/test.desc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+CORE
+My.class
+--function My.check2DCharArray --unwind 5 --max-nondet-array-length 10
+line 8 assertion at file My.java.*: FAILURE
+line 11 assertion at file My.java.*: FAILURE
```
Comment (romainbrenguier) [inline jbmc/regression/jbmc/multidimensional-array-char/test.desc]: :pick: maybe add an assertion that should succeed, like maybe for instance `assert diff < 10`

**Review by romainbrenguier — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: a5fd2df82259fd6d3161d7a67fbf147c9d856889).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/118979057

**Review by LAJW — APPROVED** [review]

**Review by owen-mc-diffblue — APPROVED** [review]

**Pull request merged**